src/cli/cmd_build_and_check.rs

```diff
@@ -234,6 +240,7 @@ pub struct CmdTest {
     #[clap(short, long)]
     debug: bool,
 
+    // todo: this is kimchi specific and it doesn't work atm. we might just remove this option at this stage?
```

Comment:
I'm fine if you remove it and disable the double generic gate by default (since kimchi is deprioritized)